main.py

```diff
@@ -1,50 +0,0 @@
-from db import database
```

Comment:
Why did you decide to move `main.py` inside `apps/`? `main.py`, `db.py` and `tags_metadata.py` feel not being in place inside `apps/` folder, because they are not apps. Can we keep them outside? I think it would be convenient to have entry point  (main.py) in the project root. Otherwise you have to update README, to ensure that instructions work